packages/sit-onyx/src/components/OnyxNavButton/OnyxNavButton.stories.ts

```diff
@@ -59,7 +59,10 @@ export const WithChildren: Story = {
   args: {
     ...Default.args,
     active: true,
-    children: () => nestedChildren.map(({ label }) => h(OnyxListItem, label)),
+    children: () =>
+      nestedChildren.map(({ label, active }) =>
+        h(OnyxNavItem, { "aria-selected": active }, [label]),
```

Comment:
We should only set the aria attribute internally and create a "real" selected/active property :)